docs/sources/investigate/index.md

```diff
@@ -0,0 +1,196 @@
+1. Optional: Select the signal you want to observe. **Full traces** are the default selection.
+1. Look for spikes or trends in the data to help identify issues.
+
+{{< admonition type="tip" >}}
+If no data or limited data appears, refresh the page. Verify that you have selected the correct data source in the Data source drop-down as well as a valid time range.
+{{< /admonition >}}
+
+## Define filters
+
+Next, refine your investigation by adding filters.
+
+Filters are available on the **Breakdown** and **Comparison** tabs.
+
+Each time you add a filter, the condition appears in the list of filters at the top of the page.
+The list of filters expands as you investigate and explore your tracing data using Explore Traces.
+
+1. Refine your investigation by adding filters.
+1. Optional: Choose one of the attributes to group by or use **Search** to locate the service.
+1. Optional: Use the tabs underneath the metrics selection to provide insights into breakdowns, comparisons, latency, and other explorations.
+1. Choose filters to hone in on the problem areas. Each filter that you select adds to the **Filter** statement at the top of the page. You can select filters on the **Comparison** and **Breakdown** tabs in the following ways:
+    * Select **Add to filters**.
+    * Select **Inspect**.
+    * Use the **Filter** bar near the top.
+
+![Change filters for your investigation](../images/explore-traces-filters.png)
+
+### Use the Breakdown tab
+
+The **Breakdown** tab highlights attributes that are correlated with the selected metric.
+When you're using **Duration** metrics, **Breakdown** orders the sequence of attributes by their average duration.
+When you select **Rate**, **Breakdown** orders the sequence of attributes by their rate of requests per second, with errors colored red.
+
+You can change the **Scope** to show **Resource** or **Span**.
+
+Using the **Group by** selector, you can group the selected metric by different attributes.
+For example, if you have selected **Errors** as a metric type and then choose the `service.name` attribute, the displayed results show the number of errors sorted by the `service.name` with the most matches.
+
+![Errors metric showing the **Breakdown** tab without filters](/media/docs/explore-traces/explore-traces-breakdown-errors-v0.9.png)
+
+The app defaults to `service.name` and displays other commonly used resource level attributes such as `cluster`, `environment`, and `namespace`.
+In the drop-down list, you can choose any resource level attribute to group by.
+
+### Modify a filter
+
+Selecting an option for a filter automatically updates the displayed data.
+If there are no matches, the app displays a “No data for selected query” message.
+
+To modify an applied filter:
+
+1. Select the filter to modify in the Filter bar.
+1. Select an option from the drop-down list.
+
+You can also click in the **Filter** bar to add filters using drop-down lists.
+
+### Remove one or more filters
```

Comment:
```suggestion
### Remove filters
```

Maybe a small nit, I think this can be more concise, and you have instructions for both in the section.